### PR TITLE
added cors

### PR DESCRIPTION
**Pull request by MikesGit138**
Needed cors to handle the request from the client side, made one of the URI global for dev purposes and should be removed upon deployment.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -5,6 +5,10 @@ import helmet from "helmet";
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  app.enableCors({
+    origin : ['*', 'http://localhost:4200'],
```
Comment (bombies) [inline src/main.ts]: Need to convert this to an environment variable.

File: src/main.ts [hunk 2]
```diff
@@ -5,6 +5,10 @@ import helmet from "helmet";
...
+    methods: ['POST', 'PUT', 'DELETE', 'GET']
```
Comment (bombies) [inline src/main.ts:10]: PATCH?

**Review by bombies — CHANGES_REQUESTED** [review]
Missing ValidationPipe global pipe. Might want to pull the most recent changes just to have the code up-to-date.

File: src/main.ts (hunk 1 above)
Comment (MikesGit138) [reply src/main.ts]: done

File: src/main.ts (hunk 2 above)
Comment (MikesGit138) [reply src/main.ts:10]: whoops yup mistake

**Review by bombies — APPROVED** [review]

**Pull request merged**